test/runtime/samples/component-binding-each-remount-keyed/_config.js

```diff
@@ -0,0 +1,39 @@
+export default {
+	html: `
+		<div data-id="1">
+			<inner>0</inner>
+			<inner>1</inner>
+		</div>
+		<div data-id="2">
+			<inner>0</inner>
+			<inner>1</inner>
+		</div>
+		<div data-id="3">
+			<inner>0</inner>
+			<inner>1</inner>
+		</div>
+	`,
+
+	ssrHtml: `
+		<div data-id="1">
+			<inner>0</inner>
+			<inner>1</inner>
+			<inner>2</inner>
+		</div>
+		<div data-id="2">
+			<inner>0</inner>
+			<inner>1</inner>
+			<inner>2</inner>
+		</div>
+		<div data-id="3">
+			<inner>0</inner>
+			<inner>1</inner>
+			<inner>2</inner>
+		</div>
+	`,
+
+	async test({ assert, component }) {
+		await component.done;
+		assert.equal(component.getCounter(), 13);
```

Comment:
Do we need to check mounted HTML as well?
like `assert.htmlEqual(target.innerHTML, '** mounted html **');`

(`test/runtime/samples/component-binding-each-remount-unkeyed/_config.js` as well)

Reply:
This can be useful, thanks!